dev-infrastructure/configurations/mvp-first-party.bicepparam

```diff
@@ -0,0 +1,5 @@
+using '../templates/first-party-identity.bicep'
+
+param kvCertOfficerManagedIdentityName = 'aro-hcp-dev-first-party'
+param globalKeyVaultName = 'aro-hcp-dev-global-kv'
```

Comment:
Why a new key vault?  Why not the service key vault?

Reply:
The service key vault will contain: 
- 1p certificate
- MSI 1p certificate
- RP TLS certificate

And all will be mounted using secrets CSI driver.  

What would be stored within the "global" key vault?  The 1p certificate is provisioned per region along with the tls certificates.

Reply:
why do we provision one key vault per region for 1p apps?  For redundancy. If a region goes down, it doesn't affect other regions.